Makefile

```diff
@@ -364,9 +364,9 @@ install-ci-chart: install-chart-prerequisite build/toolchain/bin/helm$(EXE_EXTEN
 		--set evaluator.hostName=test \
 		--set evaluator.grpcPort=50509 \
 		--set evaluator.httpPort=51509 \
-		--set open-match-core.registrationIntervalMs=100ms \
```

Comment:
and add a comment to remind future maintainers to modify the constants if they want to change the values at here

Reply:
commented that the constants much match the configuration, where they are set is less relevant.